client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -31,6 +41,58 @@ object DiagramView {
 
     private val titleVar = Var("")
 
+    def diagramControls(d: DiagramConfig): HtmlElement =
+      div(
+        cls := "flex w-full justify-around",
+        a(
+          cls := "text-4xl rounded text-center place-self-center",
+          "🔼 ",
+          onClick.map(_ => Command.MoveDiagram(d, Direction.Down)) --> Command.observer
+        ),
+        a(
+          cls := "text-4xl rounded text-center place-self-center",
+          "🔽",
+          onClick.map(_ => Command.MoveDiagram(d, Direction.Up)) --> Command.observer
+        ),
+        a(
+          cls := "text-white font-bold text-xl py-2 px-4 mx-2 rounded text-center place-self-center bg-red-500 hover:bg-red-700",
+          "Remove 💥",
+          onClick.map(_ => Command.RemoveDiagram(d)) --> Command.observer
+        )
+      )
+
+    def chartConfig(d: DiagramConfig): HtmlElement =
+      div(
+        cls := "w-full h-2/3 flex flex-col justify-between",
+        form(
+          cls := "h-full",
+          onSubmit.preventDefault.mapTo(
+            Command.UpdateDiagram(d.copy(title = titleVar.now()))
+          ) --> Command.observer,
+          p(
+            cls := "flex",
+            label(cls := "w-1/3 text-gray-50 text-xl font-bold", "Title: "),
+            input(
+              cls := "w-2/3 rounded-xl px-3 text-gray-600",
+              placeholder(s"Enter Diagram title e.g:${d.title}"),
+              controlled(
+                value <-- titleVar,
+                onInput.mapToValue --> titleVar
+              )
+            )
+          ),
+          // Using the form element's onSubmit in this example,
+          // but you could also respond on button click if you
+          // don't want a form element
```

Comment:
Not sure what the comment is supposed to say here - is that a left over from copy / paste ?